tests/v2/validation/provisioning/setup-common.go

```diff
@@ -0,0 +1,13 @@
+package provisioning
```

Comment:
rename this file `setup.go` let's not name anything with common, and avoid using underscores except for test files.